packages/restlessness-cli/rln-frontend/src/App.js

```diff
@@ -16,6 +17,7 @@ function App() {
       <Resource name="endpoints" list={EndpointsList} create={EndpointsCreate}/>
       <Resource name="models" list={ModelsList} create={ModelsCreate}/>
       <Resource name="daos" />
+      <Resource name="swagger" list={SwaggerList}/>
```

Comment:
Questo non scatena la call lato API? Non si può fare in maniera più custom?

Reply:
Farlo custom con react admin (senza usare il list) sembra troppo lungo e complicato, per ora lo lascio così ma è migliorabile sicuramente.